sqlup-mode.el

```diff
@@ -67,19 +67,12 @@
 this mode's logic will be evaluated.")
 
 (defconst sqlup-eval-keywords
-  '((postgres "EXECUTE"))
-"List of keywords introducing eval strings, organised by dialect.")
+  '((postgres "EXECUTE" "format("))
```

Comment:
No, we're missing the keyword execute here. Format is never used alone, only in the second form of execute. I had a quick look in my original branch, and the test looks like this (I hope formatting will be preserved) :

Scenario: Upcase a normal SQL keyword in a postgres-format eval string
    Given I turn on sqlup-mode
    And I start an action chain
    And I press "M-x"
    And I type "sql-highlight-postgres-keywords"
    And I execute the action chain
    And I insert "execute"
    And I type " "
    And I insert "format("
    And I insert "'select"
    And I type " "
    Then I should see "EXECUTE format('SELECT "